### ISSUE TITLE
google-translate-mf2

### ISSUE THREAD

**Issue by klavinsjanis** [translate-agent/translate#226]
Implement translation of patterns inside 'MF2' message using Google Translate.

Notes:
- Placeholders inside the patterns must not be translated!
- Other parts of the 'MF2' message must not be translated!

Examples:
```
"Hello, World!" -> "Sveika, pasaule!"
".local $var = { $anotherVar } {{Hello { $var } world}}" -> ".local $var = { $anotherVar } {{Sveika { $var } pasaule}}"
```

**Comment by janishorsts** [comment]
What this ticket is about?

**Comment by VladislavsPerkanuks** [comment]
This ticket was about:

We have mf2 message (simple/complex whatever), we send it Google Translate, and we expect to get valid mf2 back.
fixed in #230
